sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/mathExpressions.scala

```diff
@@ -170,29 +193,29 @@ case class Pi() extends LeafMathExpression(math.Pi, "PI")
 
 // scalastyle:off line.size.limit
 @ExpressionDescription(
-  usage = "_FUNC_(expr) - Returns the inverse cosine (a.k.a. arccosine) of `expr` if -1<=`expr`<=1 or NaN otherwise.",
+  usage = "_FUNC_(expr) - Returns the inverse cosine (a.k.a. arccosine) of `expr` if -1<=`expr`<=1 or NULL otherwise.",
   extended = """
     Examples:
       > SELECT _FUNC_(1);
        0.0
       > SELECT _FUNC_(2);
-       NaN
+       NULL
```

Comment:
As we already explicitly define the result value (NaN), I'm not sure we should change it like this. Compatibility issue might be arisen.